### DISCUSSION TITLE
Kafka schema naming strategy

### DISCUSSION THREAD

**Discussion by CROprogrammer** [microcks/microcks#866] [Q&A] [answered]
Which schema naming strategy does Microcks use when using it with Kafka, AsyncAPI and Schema Registry. Which naming strategy does it use for Confluent, and which for Apicurio?

**Comment by lbroudoux** [comment, marked as answer]
The `TopicRecordNameStrategy` is used for Confluent registry and the `TopicIdStrategy` is used for the Apicurio one.
